documentation/architecture-decision-records/ADR0001-i18n_library.md

```diff
@@ -0,0 +1,109 @@
+# Adopt react-intl as i18n library
+## ADR-0001
+
+* Status: proposed
+* Deciders:
+  * Articles Team
+  * WS Team
+* Date: 2019-03-18
+
+## Context and Problem Statement
+
+Psammead components need a standardised approach to localisation, including date and time formatting and string translation.
+
+## Decision Drivers
+
+* Ease of adoption
+* Documentation, community and online resources
+* Maintainability
+* Popularity / Number of Backers
+
+## Considered Options
+
+1. `react-intl`
+
+Created and maintained by yahoo.
+
+This library provides React components and an API to format dates, numbers, and strings, including pluralization and handling translations.
+
+It uses and builds on the Internationalization API built-in to JavaScript.
+
+Features:
+* Display numbers with separators.
+* Display dates and times correctly.
+* Display dates relative to "now".
+* Pluralize labels in strings
+* Support for 150+ languages.
+* Runs in the browser and Node.js.
+
+Developer Experience:
+
+The documentation is good https://github.com/yahoo/react-intl/wiki
+
+Support: 206 issues - 1 bug
+
+Licensing: Free to use under the Yahoo Inc. BSD license. See the LICENSE file for license text and copyright information.
+
+Examples:
+https://www.codeandweb.com/babeledit/tutorials/how-to-translate-your-react-app-with-react-intl
+https://medium.freecodecamp.org/internationalization-in-react-7264738274a0
+
+---
+
+2. `react-intl-universal`
+
+Created and maintained by Alibaba.
+
+It is an improvement of `react intl` in that it adds functionality for supporting translations in Vanilla JS. React-intl only does this in React Components.
+
+Features:
+* It can do what we want i.e format numbers, times and dates
+* Can run in both the browser and Node js so we will be covered when we finally work on adding SPA support )
+* Supports fetching of config files from a remote source e.g S3
+* Seems like it was built for a specific reason; supporting translations in Vanilla JS. As we’ll be using React components, I don’t think we have to worry about this.
```

Comment:
> As we’ll be using React components, I don’t think we have to worry about this.

This sentence reads oddly - as if it's a consequence of the previous sentence. I think what you're getting at is this:

```suggestion
* Seems like it was built for a specific reason; supporting translations in Vanilla JS. This doesn't apply to us, as we’ll be using React components, but it doesn't prevent us from using it.
```